acceptance/steps/helpers.go

```diff
@@ -476,15 +474,3 @@ func clientsForOperator(ctx context.Context, includeTLS bool, serviceAccountName
 		}},
 	}
 }
-
-func removeAllFinalizers(ctx context.Context, t framework.TestingT, gvk schema.GroupVersionKind) {
```

Comment:
This is unused as the upgrade tests are performed within a vCluster. (I'm pretty sure at least but I just noticed the `isolated` tag is missing in this branch)